source/_docs/configuration/remote.markdown

```diff
@@ -15,13 +15,25 @@ Remember to follow the [securing checklist](/docs/configuration/securing/) befor
 
 </div>
 
+## Setting up remote access
 <div class='note'>
-Home Assistant no longer support remote access via IP address since release 0.77, you have to use a domain name.
+Home Assistant no longer supports remote access via IP address since release 0.77, you have to use a domain name.  A Domain can be obtained cheaply from several providers. 
 </div>
 
-The most common approach is to set up port forwarding (for any port) from your router to port 8123 on the computer that is hosting Home Assistant. General instructions on how to do this can be found by searching `<router model> port forwarding instructions`. You can use any free port on your router and forward that to port 8123.
+To set up a remote instance, you will need to set up Network Address Translation, Port Forwarding on your router, to port 8123 on the computer that is hosting Home Assistant. 
 
-A problem with making a port accessible is that some Internet Service Providers only offer dynamic IPs. This can cause you to lose access to Home Assistant while away. You can solve this by using a free Dynamic DNS service like [DuckDNS](https://www.duckdns.org/).
+:::tip 
```

Comment:
This syntax doesn't work in Jekyll.

Reply:
This is supported elsewhere on the site does it require a div here?

Reply:
I see that a preview is generated.  Where can I find this preview?

Reply:
https://developers.home-assistant.io/docs/documenting/create-page#html